docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,266 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fast Fourier Transform (FFT)\n",
+    "The following gives background information that is helpful to understand how the Fast Fourier Transform (FFT) and the corresponding normalizations are defined in pyfar and how these are related to the concepts of energy and power signals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## FFT definition\n",
+    "The discrete Fourier spectrum of an arbitrary, but band-limited signal $x(n)$ is defined as\n",
+    "$$ X(k) = \\sum_{n=0}^{N-1} x(n) e^{-i 2 \\pi k \\frac{n}{N}}$$\n",
+    "using a negative sign convention in the transform kernel $e^{-i 2 \\pi k \\frac{n}{N}}$, and normalized angular frequency $\\omega_k = 2 \\pi k / N$. Analogously, the discrete inverse Fourier transform is implemented as\n",
+    "\n",
+    "$$ x(n) = \\frac{1}{N} \\sum_{k=0}^{N-1} X(k) e^{i2\\pi k \\frac{n}{N}}$$\n",
+    "\n",
+    "Real-valued time signals result in Fourier spectra with complex conjugate symmetry for negative and positive frequencies $X(k) = X(-k)^*$. In this case, the left-hand side of the spectrum is discarded, and only the right-hand side $X_R(k) = X(k)~\\forall~0 \\le k \\le \\lfloor N/2 \\rfloor$ is saved."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FFT normalizations\n",
+    "Pyfar implements five normalizations [[1]](http://www.aes.org/e-lib/browse.cfm?elib=20838) that can be applied to spectra after the DFT. The normalizations are implicitly used by the [pyfar.Signal class](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.Signal) (see `LINK-TO-AUDIO-OBJECTS-NOTEBOOK` for more information), and are available from [pyfar.dsp.fft.normalization](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.fft.html#pyfar.dsp.fft.normalization). For a Signal object `signal`, `signal.freq` contains the normalized spectrum according to `signal.fft_norm` and `signal.freq_raw` contains the raw spectrum without any normalization. **The time data (**`signal.time`**) does not change regardless of the normalization.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Definitions\n",
```

Comment:
maybe here we can do subsections with each definition with detailed explaination and example code and plot, then go to the next.